client/v2/go.mod

```diff
@@ -5,14 +5,14 @@ go 1.20
 require (
 	cosmossdk.io/api v0.7.0
 	cosmossdk.io/core v0.9.0
-	cosmossdk.io/depinject v1.0.0-alpha.3
+	cosmossdk.io/depinject v1.0.0-alpha.4
 	github.com/cockroachdb/errors v1.10.0
 	github.com/cosmos/cosmos-proto v1.0.0-beta.3
 	github.com/cosmos/cosmos-sdk v0.46.0-beta2.0.20230718211500-1d74652f6021
 	github.com/cosmos/gogoproto v1.4.10
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/exp v0.0.0-20230711153332-06a737ee72cb
+	golang.org/x/exp v0.0.0-20230809094429-853ea248256d
```

Comment:
Sneaky dependbot!

Reply:
We can only bump this one when https://github.com/cosmos/gogoproto/pull/83 is merged and we have a new tag.